notebooks/1_intelligent_agents_fall_2021/index.ipynb

```diff
@@ -0,0 +1,225 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div align=\"center\">\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <h1 style=\"font-size: 40px; margin: 10px 0;\">AI - Intelligent Agent</h1>\n",
+    "    <h1 style=\"font-size: 20px; font-weight: 400;\">Sharif University of Technology - Computer Engineering Department</h1>\n",
+    "    <br>\n",
+    "    <h4 style=\"font-size: 18px; font-weight: 400; color:#555\">Amirreza Mirzaei, Bardia Mohammadi, Sina Elahimanesh</h4>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "    <br>\n",
+    "</div>\n",
+    "<hr>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Table of contents\n",
+    "==============\n",
+    "- [Introduction](#Introduction)\n",
+    "- [Intelligent agents](#Intelligent-agents)\n",
+    "    - [Vacuum world agent example](#Vacuum-world-agent-example)\n",
+    "- [Rational agents and performance measure](#Rational-agents-and-performance-measure)\n",
+    "    - [Rationality vs perfection](#Rationality-vs-perfection)\n",
+    "    - [Autonomy](#Autonomy)\n",
+    "- [Task environment (PEAS)](#Task-environment-(PEAS))\n",
+    "    - [Types of environment](#Types-of-environment)\n",
+    "    - [PEAS example](#PEAS-example)\n",
+    "- [Type of agents](#Type-of-agents)\n",
+    "    - [Reflex agents](#Reflex-agents)\n",
+    "    - [Goal-based agents](#Goal-based-agents)\n",
+    "    - [Utility-based agents](#Utility-based-agents)\n",
+    "- [Conclusion](#Conclusion)\n",
+    "- [References](#References)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "In this notebook we will discuss the nature of intelligent agents."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Intelligent agents\n",
+    "An <b>intelligent agent</b> is anything that perceives its environment through sensors and acts upon that environment through its actuators.  \n",
+    "    we will use the term <b>percept</b> to refer to the agent's perceptual inputs at any given moment.\n",
```

Comment:
Every sentence must be started with a capital case letter and obviously, there are some similar problems in next sections. Please revise them all.